src/aat/resources/features/F-029/S-052.td.json

```diff
@@ -18,11 +18,7 @@
 	"expectedResponse": {
 		"_extends_": "Common_404_Response",
 		"body": {
-			"exception" : "[[ANYTHING_PRESENT]]",
-			"message" : "[[ANYTHING_PRESENT]]",
-			"details" : null,
-			"callbackErrors" : null,
-			"callbackWarnings" : null
-		}
+      "_guid_": "F-029_ResponseData"
```

Comment:
Started putting "_guid_" instead of "_extends", needs to be changed in S-053 too